### PR TITLE
Add project authorship/citation/funding metadata

### PR DESCRIPTION
**Pull request by MattF-NSIDC**
TODO:

- [x] Add orcid
- [x] Add other authors
- [ ] Review abstract
- [ ] Review keywords
- [ ] Add DOI (chicken-and-egg problem; we need to do a release to trigger Zenodo to generate a DOI, then we can update this file)
- [x] Add grant number
- [ ] Consider other references. Are there other DOIs that are related to this work?
- [x] Consider how to automate version bumps in `CITATION.cff`. `bump-my-version`?

### CODE REVIEW

**Comment by MattF-NSIDC** [comment]
@betolink I flipped the Zenodo switch; you should be able to see this if you log in on your GitHub account. Once we merge this PR and make a release, we should have our DOI.

https://zenodo.org/account/settings/github/repository/nsidc/earthaccess

**Comment by MattF-NSIDC** [comment]
@jrbourbeau would you mind pushing a commit to this PR adding your preferred email in `pyproject.toml`?

**Comment by mfisher87** [comment]
Should we also update the contributing document to request people to add themselves to the authors list when they make a contribution?

**Comment by mfisher87** [comment]
@betolink can you add your orcid and other authors to this PR?

**Comment by betolink** [comment]
also if we can have all the contributors'  ORCID numbers that'd be great! @jhkennedy @JessicaS11 @asteiker @cgentemann @jroebuck932 et al.

**Comment by jhkennedy** [comment]
@betolink here's my ORCID: [0000-0002-9348-693X](https://orcid.org/0000-0002-9348-693X)

**Comment by jules32** [comment]
Hi @betolink !

Award number is NASA ROSES 80NSSC21K0564 Award # 20-TWSC20-2-0003
My ORCID: https://orcid.org/0000-0003-1682-3872

**Comment by jrbourbeau** [comment]
> @jrbourbeau would you mind pushing a commit to this PR adding your preferred email in pyproject.toml?

Apologies for the delayed response. Just pushed my email 👍

**Comment by andypbarrett** [comment]
@betolink here's my ORCID 0000-0003-4394-5445

**Comment by asteiker** [comment]
@betolink Here's my ORCID: 0000-0002-3039-0260 Thanks!

**Comment by mfisher87** [comment]
@jrbourbeau do you have an ORCiD?

**Comment by jrbourbeau** [comment]
Ah, yeah I forgot I had one (had to dust it off 🙂 ) 0000-0003-2164-7789

**Comment by mfisher87** [comment]
Thanks, James :)

**Comment by mfisher87** [comment]
@betolink this is ready for a  review, can you take a final peek keeping in mind the checklist in the description?

**Review by betolink — DISMISSED** [review]
@mfisher87 circling back, the abstract is WIP and we can take a look at it after the UWG, I think we covered the core items on the checklist. Thanks for updating this PR, should we merge and release v0.5.4?

**Comment by mfisher87** [comment]
I think this is good to merge! :rocket: Would you like to do the release and in the process validate the new release instructions? :)

I pushed two more commits to fix a mistake and add an automation I missed.

**Comment by mfisher87** [comment]
Woops... missed one more thing.

**Comment by battistowx** [comment]
@betolink Here's mine: https://orcid.org/0000-0002-9608-3634

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,16 +68,44 @@ scripts/format.sh
...
-   variables, function names, decorators, etc..
-4. Increase the version numbers in any examples files and the README.md to the new version that this
-   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
-5. You may merge the Pull Request in once you have the sign-off of another developers, or if you
-   do not have permission to do that, you may request the reviewer to merge it for you.
+2. Ensure your code is formatted properly following this document
+3. Update the documentation and the README.md with details of changes to the interface,
+   this includes new environment variables, function names, decorators, etc.
+3. Update `CHANGELOG.md` with details about your change in a section titled
+   `Unreleased`. If one does not exist, please create one.
```
Comment (mfisher87) [inline CONTRIBUTING.md:78]: Feels like we should have a PR template with a checklist for things like this!

**Comment by mfisher87** [comment]
If you didn't get added in this PR, please open another PR to add yourself following this example. It's merge time!

**Review by mfisher87 — APPROVED** [review]

**Pull request merged**